service/controller.go

```diff
@@ -95,6 +98,8 @@ const (
 	removeModeOnlyMe                    = "ONLY_ME"
 	sioGatewayNotFound                  = "Not found"
 	sioGatewayVolumeNotFound            = "Could not find the volume"
+	sioGatewayNFSExportNotFound         = "couldn't find NFS export"
```

Comment:
If this error code is not being used, we can remove its instance from here.

Reply:
done